packages/orchestration/src/examples/auto-stake-it.contract.js

```diff
@@ -2,18 +2,57 @@ import {
   EmptyProposalShape,
   InvitationShape,
 } from '@agoric/zoe/src/typeGuards.js';
-import { M } from '@endo/patterns';
+import { makeTracer } from '@agoric/internal';
+import { M, mustMatch } from '@endo/patterns';
 import { prepareChainHubAdmin } from '../exos/chain-hub-admin.js';
 import { preparePortfolioHolder } from '../exos/portfolio-holder-kit.js';
 import { withOrchestration } from '../utils/start-helper.js';
-import { prepareStakingTap } from './auto-stake-it-tap-kit.js';
 import * as flows from './auto-stake-it.flows.js';
+import { ChainAddressShape } from '../typeGuards.js';
+
+const trace = makeTracer('AutoStakeIt');
 
 /**
+ * @import {GuestInterface} from '@agoric/async-flow';
  * @import {Zone} from '@agoric/zone';
+ * @import {IBCChannelID, VTransferIBCEvent} from '@agoric/vats';
+ * @import {TargetApp} from '@agoric/vats/src/bridge-target.js';
+ * @import {ChainAddress, CosmosValidatorAddress, Denom} from '@agoric/orchestration';
+ * @import {TypedPattern} from '@agoric/internal';
+ * @import {CosmosOrchestrationAccount} from '../exos/cosmos-orchestration-account.js';
+ * @import {LocalOrchestrationAccount} from '../exos/local-orchestration-account.js';
  * @import {OrchestrationPowers, OrchestrationTools} from '../utils/start-helper.js';
  */
 
+/**
+ * @typedef {{
+ *   stakingAccount: GuestInterface<CosmosOrchestrationAccount>;
+ *   localAccount: GuestInterface<LocalOrchestrationAccount>;
```

Comment:
If we use `OrchestrationAccount<any>`, which the flow is expecting, we see `Argument of type '<OrchestrationAccount<any>>' is not assignable to parameter of type 'Passable'.` on L105.